index.js

```diff
@@ -123,6 +124,47 @@ function fastifyMultipart (fastify, options, done) {
         req.body = body
       }
     })
+
+    // The following is not available on old Node.js versions
+    // so we must skip it in the test coverage
+    /* istanbul ignore next */
+    if (globalThis.FormData) {
+      fastify.decorateRequest('formData', async function () {
```

Comment:
Would you consider this a breaking change for the same reason as in https://github.com/fastify/fastify-env/pull/183#discussion_r1590240512?

That `formData` might already be decorated since it's a common name?

Reply:
Done